plugins/generator-1.18.2/forge-1.18.2/procedures/entity_get_spawn_x.java.ftl

```diff
@@ -0,0 +1,10 @@
+(new Object() {

```

Comment:
This could probably be reworked into two ternary operators instead of creating an anonymous object